modules/standard/Curl.chpl

```diff
@@ -24,21 +24,21 @@ Simple support for many network protocols with libcurl
 This module provides support for libcurl, enabling Chapel programs
 to work with many network protocols. 
 
-The `curl homepage <http://curl.haxx.se/libcurl/>`_ describes libcurl thus::
+The `curl homepage <https://curl.haxx.se/libcurl/>`_ describes libcurl thus::
 
  libcurl is a free and easy-to-use client-side URL transfer library, supporting
- DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP, LDAPS, POP3,
+ DICT, FILE, FTP, FTPS, Gopher, https, httpsS, IMAP, IMAPS, LDAP, LDAPS, POP3,
```

Comment:
Typo on the extra s here?  I haven't heard of httpsS so it might just be my ignorance